hamcrest-library/src/main/java/org/hamcrest/number/BigDecimalCloseTo.java

```diff
@@ -13,7 +13,7 @@
   private final BigDecimal value;
 
   public BigDecimalCloseTo(BigDecimal value, BigDecimal error) {
-      this.delta = error;
+      this.delta = error.abs();
```

Comment:
is error null checked 1st, or should it be null checked?

Reply:
@nhojpatrick jap, checking this for null would be better.

Reply:
@StarBax89 this file now lives at `./hamcrest/src/main/java/org/hamcrest/number/BigDecimalCloseTo.java`

if you rebase from the latest master, it should resolve the conflicts and I can merge in, e.g. `git fetch origin && git rebase origin/master`